servers/jax-rs-tests/src/test/java/org/projectnessie/jaxrs/TestJerseyRestNaiveClientInMemory.java

```diff
@@ -48,25 +46,22 @@
 class TestJerseyRestNaiveClientInMemory extends AbstractTestJerseyRest {
 
   @Override
-  protected void init(NessieApiV1 api, @Nullable HttpClient.Builder httpClient) {
-    assumeThat(httpClient).isNotNull();
-
+  protected void initApi(URI nessieApiUri) {
     // Intentionally remove the `Accept` header from requests.
     // Service endpoints should declare the content type for their return values,
     // which should allow the Web Container to properly format output even in the absence
     // of `Accept` HTTP headers.
     RequestFilter noAcceptFilter = context -> context.removeHeader(HEADER_ACCEPT);
-    httpClient.addRequestFilter(noAcceptFilter);
 
-    api =
+    NessieApiV1 api =
         HttpClientBuilder.builder()
             // Abuse the authentication callback a bit to inject the noAcceptFilter into
             // the java client.
             .withAuthentication(
                 (HttpAuthentication) client -> client.addRequestFilter(noAcceptFilter))
-            .withUri(httpClient.getBaseUri())
```

Comment:
we only used the base uri from the passed in `httpClient`... so we should instead override `initApi(URI)`